src/Components/GameOverMenu.js

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { useState } from "react";
+import * as myConstants from "./Constants";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { createTheme } from '@mui/material/styles';
+import { ThemeProvider } from "@mui/material";
+import { Stack } from "react-bootstrap";
+
+function GameOverMenu(props) {
+  const [gameScoreSaved, setGameScoreSaved] = useState(false);
+
+  const clickContinue = () => {
+    props.onClick();
+  };
+
+  const saveScore = () => {
+    let lanesAsString = String(props.end.lanesNum);
+
+    let scoreboard = JSON.parse(localStorage.getItem("scoreboard"));
+    if (!scoreboard) scoreboard = {};
+
+    let currentScores = scoreboard[lanesAsString];
+    if (!currentScores) currentScores = [];
+
+    currentScores.push({
+      username: props.end.username,
+      lanesNum: props.end.lanesNum,
+      score: props.end.score,
+      causeOfDeath: props.end.message,
+      discMode: myConstants.useDiscoMode,
+    });
+
+    currentScores.sort((a, b) => b.score - a.score);
+
+    if (currentScores.length > myConstants.maximumNumberOfScores)
+      currentScores.pop();
+
+    scoreboard[lanesAsString] = currentScores;
+
+    localStorage.setItem("scoreboard", JSON.stringify(scoreboard));
+
+    setGameScoreSaved(true);
+  };
+
+  const displayScores = () => {
+    let lanesAsString = String(props.end.lanesNum);
+    let scoreboard = JSON.parse(localStorage.getItem("scoreboard"));
+
+    if (!scoreboard[lanesAsString]) return <li>no current scores</li>;
+
+    let arr = [];
+
+    for (let i = 0; i < scoreboard[lanesAsString].length; i++) {
+      arr.push(
+        <li key={i} >
+            <div style={{backgroundColor: "white", borderRadius:"1vh"}}>
+                <h1 style={{color: "black",fontSize:"2vh"}}>{`${scoreboard[lanesAsString][i].username}`}</h1>
+                <p style={{color: "black",fontSize:"1.5vh", padding:"0.5vh" }}>
+                    {`score: ${scoreboard[lanesAsString][i].score}, cause of death:${scoreboard[lanesAsString][i].causeOfDeath}`}
+                </p>
+            </div>
+        </li>
+      );
+    }
+    return arr;
+  };
+  const theme = createTheme({
```

Comment:
this is the same theme as in start menu. Use the theme provider of material properly. It need's to wrap the entire project and have themeing everywhere.